docs/proposals/automatically-propagate-relevant-resources/README.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Automatically Propagate Relevant Resources
+authors:
+- "@mrlihanbo"
+reviewers:
+- "@TBD"
+approvers:
+- "@TBD"
+
+creation-date: 2021-09-06
+
+---
+
+# Automatically Propagate Relevant Resources
+
+## Summary
+
+Secrets or ConfigMaps can be mounted as data volumes or exposed as environment variables to be used by a container in a Pod.
+Meanwhile, users can use a Service to expose the application running on a set of Pods.
+ 
+Thus, it is necessary to deploy the relevant resource into the same member clusters when propagating a Pod by Karmada, i.e, 
+Pod dependencies(ConfigMaps and Secrets) and dependents(Services and Ingresses) should "follow" the delegate pods.
+
+Usually，users don't create Pods directly. Instead, create them using workload resources such as Deployment, StatefulSet or Job.
```

Comment:
This symbol `Usually，` doesn't look well formatted.

Reply:
Nice finding!